### DISCUSSION TITLE
Theme

### DISCUSSION THREAD

**Discussion by Stoufs** [rejetto/hfs#469] [General]
Sorry if double post but how do i change my theme to something like this?  Is it an addon or self written?

https://fileserver.ph5hp.nl

**Comment by PH5HP** [comment]
Hi Stoufs,

You need some knowledge about CSS and how it works with F12 in Chrome. F12 in Chrome shows you information about how a website looks in the browser. In other words: how it is done. My fileserver is a combination of addons and selfwritten.

Fileserver.ph5hp.nl is my fileserver and I added some CSS and HTML in sections under the CUSTOM HTML section in the HFS main menu.

For example:

To change the button color
------------------------------

<style>

/* button color */
button {
    background-color: #0000A0;
    color: #ffffff;
}

</style>

-------------------------------

Everything from <style> to </style> has been added to the BEFORE HEADER section under CUSTOM HTML in the main menu of HFS. Lines starting with /* and ending with */ are just text to explain what it does. Within Google Chrome you can experiment with the color etc to see what it does.

See the screenshot attached. Left side my fileserver and on the right side the screen you get if you press F12. Look at the row highlighted there. Beginning with 'button title'.  To go to a specific item of which you want to know the parameters of then press right mouse button when you are standing on it and choose for inspect. 

On the right side you see a lot of items in white and light blue. Somewhere in the middle there you see:

button {
  background-color: #0000A0;
  color: #ffffff;
}

That determines the font color and the background color of the button. If you change that in the F12-screen then you see instantly the result. If you are done then you have to copy and paste it into the right section of CUSTOM HTML in the main menu of HFS.

Feel free to ask me about this. No problem.

Kind regards,
Henk - fileserver.ph5hp.nl
![image](https://github.com/rejetto/hfs/assets/140949730/9af2638c-8b8c-4a0a-8cfd-dd7a874919f7)
